README.md

```diff
@@ -205,7 +205,7 @@ Other Style Guides
       value: 1,
 
       addValue: function (value) {
-        return atom.value + value;
+        return this.value + value;
```

Comment:
this is an incorrect change, since `this` will depend on the way `addValue` is called, but hardcoding `atom` will not.